jenkins/deployment/community/cortx-community-deploy.groovy

```diff
@@ -15,210 +14,204 @@ pipeline {
+                    export CORTX_RE_BRANCH=${CORTX_RE_BRANCH}
+                    export CORTX_TAG=${CORTX_TAG}
+                    sleep 120
+                    export PRIMARY_PUBLIC_IP=$(cat ip_public.txt | jq '.[0]'| tr -d '",[]')
+                    ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@${PRIMARY_PUBLIC_IP} "export CORTX_RE_BRANCH=$CORTX_RE_BRANCH; git clone $CORTX_RE_REPO -b $CORTX_RE_BRANCH; pushd /home/centos/cortx-re/solutions/community-deploy; time sudo ./build-cortx.sh"
                     popd
             '''
             }
         }
-
-        stage ('Setup K8s cluster on EC2') {
+        stage('Setup K8s cluster') {
             steps {
                 script { build_stage = env.STAGE_NAME }
-                sh label: 'setting up K8s cluster on EC2', script: '''
-                pushd solutions/community-deploy/cloud/AWS                
-                    ssh -i cortx.pem -o StrictHostKeyChecking=no centos@"${AWS_IP}" "pushd /home/centos/cortx-re/solutions/kubernetes && sudo ./cluster-setup.sh true"
+                sh label: 'Setting up K8s cluster on EC2', script: '''
+                pushd solutions/community-deploy/cloud/AWS
+                    export WORKER_IP=$(cat ip_public.txt | jq '.[1]','.[2]' | tr -d '",[]')
+                    export CORTX_SERVER_IMAGE=$HOST1:8080/seagate/cortx-rgw:2.0.0-0
+                    export CORTX_DATA_IMAGE=$HOST1:8080/seagate/cortx-data:2.0.0-0
+                    export CORTX_CONTROL_IMAGE=$HOST1:8080/seagate/cortx-control:2.0.0-0
+                    ssh -i cortx.pem -o StrictHostKeyChecking=no centos@${PRIMARY_PUBLIC_IP} "sudo -- sh -c 'pushd /home/centos/cortx-re/solutions/kubernetes && ./cluster-setup.sh true && sed -i 's,cortx-docker.colo.seagate.com,${HOST1}:8080,g' /etc/docker/daemon.json && systemctl restart docker && sleep 240'"
+                    for wp in $WORKER_IP;do ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@${wp} "sudo -- sh -c 'sed -i 's,cortx-docker.colo.seagate.com,${HOST1}:8080,g' /etc/docker/daemon.json && systemctl restart docker && sleep 240 && docker pull ${CORTX_SERVER_IMAGE} && docker pull ${CORTX_DATA_IMAGE} && docker pull ${CORTX_CONTROL_IMAGE}'";done
                     popd
-                '''
+            '''
             }
         }
-
-        stage ('Deploy 1N cortx cluster') {
+        stage('Deploy multi-node cortx cluster') {
             steps {
                 script { build_stage = env.STAGE_NAME }
-                sh label: 'Deploying 1N cortx cluster on EC2', script: '''
+                sh label: 'Deploying multi-node cortx cluster and pull locally generated cortx images on worker nodes', script: '''
```

Comment:
`Pull locally generated cortx images on worker nodes` code is in the previous stage at https://github.com/mukul-seagate11/cortx-re-1/blob/communitybuild-multinode-auto/jenkins/deployment/community/cortx-community-deploy.groovy#L134  
It should be moved here to align with the stage name.